src/main.css

```diff
@@ -0,0 +1,67 @@
+@import url('https://fonts.googleapis.com/css?family=Alice|Changa+One');
+
+body {
+        background-image: url(../pics/hawaii-beach.png);
+        background-repeat: no-repeat;
+        background-size: cover;
+        margin: auto;
+        text-align: center;
+        /*position: fixed;
```

Comment:
If the code is not being used dont leave it in the PR! Also, this shouldn't work because your ```body``` selector is missing a closing ```}```.